(1) Community Patch/Kit/CPK.lua

```diff
@@ -28,7 +28,7 @@
 
 	local EmulateInclude = (function()
 		local function GetAvailablePaths()
-			local list = IncludeFileList --[[ @as string[] | nil ]]
```

Comment:
Spaces were added because editor freaked out when it was just `string[]` like in this cast `--[[@as string[]]]`, because it messed lua comment and added extra `]` after it